### ISSUE TITLE
dotnet migrate should add trailing new line at the end of generated proj file

### ISSUE THREAD

**Issue by piotrpMSFT** [dotnet/cli#4314]
Currently:
```
...
  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" /> 
</Project>
 \ No newline at end of file 
```

**Comment by brthor** [comment]
We could open the file and add a newline but we're using the msbuild api to construct and save the file here. If this is a general request for msbuild files, I think the fix for this should go somewhere in the msbuild construction api.

https://github.com/Microsoft/msbuild/blob/xplat/src/XMakeBuildEngine/Construction/ProjectRootElement.cs#L1671

@rainersigwald is this something that seems reasonable, what's your opinion?

**Comment by rainersigwald** [comment]
Are you using the new preserve-whitespace approaches (https://github.com/Microsoft/msbuild/pull/1036)? If those eat a trailing newline we should definitely fix it. But our old behavior is to just let `System.Xml` do what it wants, which isn't easy to change.

**Comment by brthor** [comment]
@rainersigwald Let me give that a try, do you happen to know if that got merged into xplat?

**Comment by rainersigwald** [comment]
@brthor If I'm reading the history graph right it should be in the '41 package and up.

**Comment by livarcocc** [comment]
Blocked by https://github.com/Microsoft/msbuild/issues/1228.
